src/app/pages/menu/contact/contact.page.ts

```diff
@@ -1,36 +1,44 @@
 import { Component, OnInit } from '@angular/core';
 import { Router, NavigationExtras, ActivatedRoute } from '@angular/router';
 
-import { UserService } from 'src/app/services/user/user.service';
+import { UserService, User } from 'src/app/services/user/user.service';
 
 @Component({
   selector: 'app-contact',
   templateUrl: './contact.page.html',
   styleUrls: ['./contact.page.scss'],
 })
 export class ContactPage implements OnInit {
-  contact;
```

Comment:
Ok, so up until now Contact page was just kind of hanging out there not used. Just so we're on the same page, this page will be used to view the content of held cards in the rolodex. Is that the direction your code is pointed?

Reply:
Yes. A new dedicated page could be useful but I honestly forget the original purpose of this page and I did not want to use the `user-details` page for card details.